reporter/otlp_reporter.go

```diff
@@ -546,13 +546,15 @@ func (r *OTLPReporter) getProfile() (profile *profiles.Profile, startTS, endTS u
 
 		// Walk every frame of the trace.
 		for i := range traceInfo.frameTypes {
+			frameAttributes := addProfileAttributes(profile, []attrKeyValue{
```

Comment:
This for allow "abort-marker" (`abortFrameName`) to be a value that the agent sends with `profile.frame.type`, but there's no semantic convention for it.

Reply:
@christos68k Does it make sense to create a separate issue for this, so we can merge this PR. Without it, the devfiler won't work with the latest agent.